src/services/locatieService/cityservice.ts

```diff
@@ -2,15 +2,25 @@ import City from '@/classes/City';
 import http from '@/services/http';
 
 export default class CityService {
-    public async post(cityModel): Promise<City> {
-        const response = await http.post(`/api/locations/cities`, cityModel);
-        return response.data;
-     }
+  public async post(cityModel): Promise<City> {
+    const response = await http.post(`/api/locations/cities`, cityModel);
+    return response.data;
+  }
 
-     public async getAll(): Promise<Array<City>> {
-        const response = await http.get(`/api/locations/cities`);
-        return response.data;
-     }
+  public async getAll(): Promise<Array<City>> {
+    const response = await http.get(`/api/locations/cities`);
+    return response.data;
+  }
+
+  public async getById(id: string): Promise<City> {
+    const response = await http.get(`/api/locations/cities/${id}`);
+    return response.data;
+  }
+
+  public async updateCity(cityModel): Promise<City> {
```

Comment:
zou de id hier uit citymodel halen en apart geven zoals je bij building hebt gedaan. dan is het ook mogelijk op bij de post een CityRequest te specificeren.